lang/cy.js

```diff
@@ -0,0 +1,23 @@
+/* eslint quotes: 0 */
+
+export default {
+	"appliedFilters": "Hidlyddion a Gymhwyswyd:",
+	"noActiveFilters": "Dim hidlyddion gweithredol",
+	"filterRemoved": "Mae’r Hidlydd {filterText} wedi’i dynnu",
+	"clearFilters": "Clirio hidlyddion",
+	"allFiltersRemoved": "Tynnwyd pob hidlydd",
+	"categoryTitleMultiple": "{title} ({numSelected})",
+	"searchBy": "Chwilio yn ôl {category}",
+	"clear": "Clirio",
+	"filter": "Hidlo",
+	"filterBy": "Hidlo yn ôl",
+	"filterMultiple": "Hidlo: {numOptions} Hidlyddion",
+	"filterSingle": "Hidlo: 1 Hidlydd",
+	"more": "mwy",
```

Comment:
I wonder if the More langterms (which I'm guessing you got from `core`) should be capitalized?

Reply:
Also noticed this in `da.js` and `de.js`

Reply:
Right -- updated.